internal/miner/miner.go

```diff
@@ -0,0 +1,22 @@
+package miner
+
+import (
+	"github.com/blinklabs-io/bluefin/internal/config"
+	"github.com/blinklabs-io/bluefin/internal/logging"
+)
+
+type Miner struct {
+	Config *config.Config
+	Logger *logging.Logger
+}
+
+func NewMiner(cfg *config.Config, logger *logging.Logger) *Miner {
```

Comment:
New() and use config.GetConfig() and logging.GetLogger() so they don't need to be passed into the function.